2021-10-04 19:20:44,908 INFO    MainThread:108748 [wandb_setup.py:_flush():69] setting env: {}
2021-10-04 19:20:44,908 INFO    MainThread:108748 [wandb_setup.py:_flush():69] setting login settings: {}
2021-10-04 19:20:44,908 INFO    MainThread:108748 [wandb_init.py:_log_setup():342] Logging user logs to /home/jet/Sandboxes/railway_drone/wandb/run-20211004_192044-2s466vbh/logs/debug.log
2021-10-04 19:20:44,908 INFO    MainThread:108748 [wandb_init.py:_log_setup():343] Logging internal logs to /home/jet/Sandboxes/railway_drone/wandb/run-20211004_192044-2s466vbh/logs/debug-internal.log
2021-10-04 19:20:44,908 INFO    MainThread:108748 [wandb_init.py:init():375] calling init triggers
2021-10-04 19:20:44,908 INFO    MainThread:108748 [wandb_init.py:init():380] wandb.init called with sweep_config: {}
config: {'num_envs': 10, 'max_steps': 2000, 'buffer_size': 100, 'buffer_size_debug': 100, 'batch_size': 300, 'start_epoch': 0, 'epochs': 10000, 'pretrain_epochs': 3, 'transitions_per_epoch': 100, 'repeats_per_buffer': 2, 'scheduler_steps': 5, 'scheduler_gamma': 0.5, 'starting_LR': 0.001, 'reg_lambda': 0.0, 'weights_directory': 'weights', 'optim_weights_directory': 'optim_weights', 'tracknet_version': '_7', 'tracknet_number': 0, 'epoch_interval': 1, 'batch_interval': -1, 'debug': True, 'wandb': True, 'device': device(type='cuda', index=0), 'iter_per_epoch': 66, 'step_sched_num': 132000.0}
2021-10-04 19:20:44,908 INFO    MainThread:108748 [wandb_init.py:init():424] starting backend
2021-10-04 19:20:44,908 INFO    MainThread:108748 [backend.py:_multiprocessing_setup():70] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-10-04 19:20:44,914 INFO    MainThread:108748 [backend.py:ensure_launched():135] starting backend process...
2021-10-04 19:20:44,918 INFO    MainThread:108748 [backend.py:ensure_launched():139] started backend process with pid: 108766
2021-10-04 19:20:44,919 INFO    MainThread:108748 [wandb_init.py:init():429] backend started and connected
2021-10-04 19:20:44,921 INFO    MainThread:108748 [wandb_init.py:init():477] updated telemetry
2021-10-04 19:20:44,921 INFO    MainThread:108748 [wandb_init.py:init():500] communicating current version
2021-10-04 19:20:45,447 INFO    MainThread:108748 [wandb_init.py:init():505] got version response upgrade_message: "wandb version 0.12.3 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-10-04 19:20:45,447 INFO    MainThread:108748 [wandb_init.py:init():513] communicating run to backend with 30 second timeout
2021-10-04 19:20:45,700 INFO    MainThread:108748 [wandb_init.py:init():540] starting run threads in backend
2021-10-04 19:20:46,523 INFO    MainThread:108748 [wandb_run.py:_console_start():1584] atexit reg
2021-10-04 19:20:46,523 INFO    MainThread:108748 [wandb_run.py:_redirect():1458] redirect: SettingsConsole.REDIRECT
2021-10-04 19:20:46,523 INFO    MainThread:108748 [wandb_run.py:_redirect():1463] Redirecting console.
2021-10-04 19:20:46,524 INFO    MainThread:108748 [wandb_run.py:_redirect():1519] Redirects installed.
2021-10-04 19:20:46,524 INFO    MainThread:108748 [wandb_init.py:init():565] run started, returning control to user process
2021-10-04 19:27:47,620 INFO    MainThread:108748 [wandb_run.py:_atexit_cleanup():1554] got exitcode: 0
2021-10-04 19:27:47,620 INFO    MainThread:108748 [wandb_run.py:_restore():1526] restore
